features/flows/elastic/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -62,12 +55,6 @@
             <cm:property name="alwaysUseRawForQueries" value="true" />
             <cm:property name="timeRangeDurationAggregateThresholdMs" value="120000" /> <!-- 2 minutes -->
             <cm:property name="timeRangeEndpointAggregateThresholdMs" value="604800000" /> <!-- 7 days -->
-
-            <!-- Enrichment settings -->
-            <cm:property name="clockSkewCorrectionThreshold" value="0" />
-
-            <!-- Flow Mangling -->
-            <cm:property name="mangleScriptPath" value="" />
```

Comment:
same for this; if we're losing tunable settings, we should describe how to replace their functionality, or why you shouldn't need to anymore

Reply:
Same as above